frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -53,8 +54,6 @@ export function getMetabaseSdkCssVariables(theme: MantineTheme) {
  **/
 function getSdkDesignSystemCssVariables(theme: MantineTheme) {
   return css`
-    --mb-default-font-family: "${theme.fontFamily}";
```

Comment:
`theme.fontFamily` defaults to `fontFamily: "var(--mb-default-font-family), sans-serif",` so this didn't work